custom/icds/messaging/indicators.py

```diff
@@ -316,45 +293,45 @@ def get_messages(self, language_code=None):
             return []
 
 
+def _get_last_submission_dates(awc_ids):
```

Comment:
This would lot of requests to Citus, right? Similar to `get_awws_in_vhnd_timeframe` that caches list of all awc_ids that satisfy some condition, we could make `get_awws_in_last_30_days` and `get_awws_in_last_one_week`, which would hit Citus once and put it in cache. 

If this is not a concern, then it's okay.